### PR TITLE
Added logger

### PR DESCRIPTION
**Pull request by rafaelsideguide**

### CODE REVIEW

File: apps/api/src/lib/logger.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    const messageLevelIndex = levels.indexOf(level);
+
+    if (currentLevelIndex >= messageLevelIndex) {
+      const color = Logger.colors[level];
+      console[level.toLowerCase()](color, `[${new Date().toISOString()}]${level} - ${message}`);
+
+      // if (process.env.USE_DB_AUTH) {
+      // save to supabase? another place?
+      // supabase.from('logs').insert({ level: level, message: message, timestamp: new Date().toISOString(), success: boolean });
+      // }
```
Comment (mogery) [inline apps/api/src/lib/logger.ts:31]: we don't need this -- the logs we'll save into DB will be very targeted and rich with structured data

**Review by mogery — APPROVED** [review]
LGTM

**Comment by nickscamara** [comment]
@rafaelsideguide is this ready for review?

**Comment by rafaelsideguide** [comment]
@nickscamara all good now. There are still some improvs but its good for now

**Pull request merged**